rakelib/lint.rake

```diff
@@ -0,0 +1,28 @@
+namespace :lint do
```

Comment:
Stencilswiftkit and this repo match fairly well. The templates repo has some small differences, the swiftgen one even more. So no idea how far we can go in unifying these